### DISCUSSION TITLE
Turbopack with Next.js 13.3.0 got 1 second slower than that with Next.js 13.2.4

### DISCUSSION THREAD

**Discussion by sapphi-red** [vercel/turborepo#4633] [Show and tell]
I wasn't sure where to report this so I'll use a discussion.

Recently I took a benchmark with Turbopack using Next.js 13.2.4 and 13.3.0.
I found Turbopack with Next.js 13.3.0 was 1 second slower than the previous one. (accurately, 0.752 second)
The methods I used are written here: https://gist.github.com/sapphi-red/25be97327ee64a3c1dce793444afdf6e

I don't know if this counts as a performance degradation but I'm reporting this for your awareness

**Comment by anthonyshew** [comment]
Appreciate you reporting this. 🙏 Thank you for using Turbopack!

We're much further along with Turbopack today and seeing much faster compilation times. You can stay up to date with [areweturboyet.com](https://areweturboyet.com) to see how close we are to all tests passing for the development server!
